MTM-Backend/src/routes/v1/user/user.router.ts

```diff
@@ -25,14 +25,14 @@ userRouter.get(
 );
 
 /**
- * get a user by id
+ * get a user by email
  */
 userRouter.get(
-  "/v1/:id",
+  "/v1/:email",
```

Comment:
let's do query for get method instead of param (in the form of "v1?email= & organization = ")

Reply:
consolidated get user endpoint with query parameters